spec/models/family_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Family do
+  describe "validations" do
+    subject { Family.new(last_name: nil) }
+
+    context "when creating a new family with no last_name is not valid" do
+      it { should_not allow_value(nil).for(:last_name)}
+    end
+
+    context "when creating a new family with no capability is not valid" do
+      it { should_not allow_value(nil).for(:capability)}
+    end
+
+    context "when creating a new family that is valid it is saved" do
+      it { expect(build(:family)).to be_valid }
+    end
```

Comment:
There is no value in the above spec, which checks that the factory is doing its job. 
We should be conscious of the specs we add, to keep the test suite lean and running fast.

Reply:
Ok, I will delete it. Thanks!